src/pages/meal-plan/meal-plan-page.tsx

```
@@ -103,17 +117,313 @@ const MealPlanPage = () => {
+  return (
+    <>
+      <div className="pt-[33px] pb-[11px]">
+        <div className="flex px-[10px] py-1 rounded-lg text-4 gap-2 bg-white">
+          <p className="font-semibold text-gray-700 text-[16px]">{`${month}월 ${weekKor}`}</p>
+          <p className="font-medium text-gray-500 text-[16px]">
+            {listHeaderDate}
+          </p>
+        </div>
+      </div>
+      <div className="flex flex-col gap-3 pb-30">
+        {futureDays.map((day) => {
```

Biome (2.4.4)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)